source/_integrations/media_source.markdown

```diff
@@ -37,7 +37,8 @@ If you are a Home Assistant Core user, the default directory called is called
 `media` under the configuration path (where your `configuration.yaml` is located).
 
 Files served from `media` are protected by Home Assistant authentication
-unlike those served from `www`.
+unlike those served from `www`. If other `media_dirs` are not declared you need
+to use `/media/local` path for example in companion app notification.
```

Comment:
I'm not sure how this is related to the rest of the sentence?

Doesn't this fit better at line 29?

Reply:
You're right. Changed it.